### ISSUE TITLE
NTR: Red blood cells high in CSF (RBC) (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8455]
New term request Red blood cells high in CSF (RBC) (perlma_100809051146): Cerebrospinal fluid erythrocytes  UMLS: 5239197

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000202
- label: Cerebrospinal fluid erythrocytes
- parents: [Abnormal cerebrospinal fluid morphology (HP:0002921)](https://hpo.jax.org/app/browse/term/HP:0002921)
- pmid: PMID:32319250
- def: The presence of red blood cells (RBCs) in the cerebrospinal fluid (CSF). The CSF normally does not contain RBCs (erythrocytes).
- comment: Erythrocytes may be seen in CSF following a traumatic lumbar tap, but in such cases RBC counts typically decrease in sequential samples. Penetration of blood into the subarachnoid space and/or CSF may be due to trauma, a ruptured aneurysm, intracerebral hemorrhage, or in connection with a tumor.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8455
- synonyms:
